src/jdk.jdeps/share/classes/com/sun/tools/javap/LocalVariableTableWriter.java

```diff
@@ -50,19 +46,21 @@
 public class LocalVariableTableWriter extends  InstructionDetailWriter {
```

Comment:
nit: two spaces between `extends` and `InstructionDetailWriter`

Reply:
fixed, thanks